README.md

```diff
@@ -1,4 +1,15 @@
 appengine-maven-archetypes-java
 ===============================
 
-App Engine Maven Archetypes
+* appengine-standard-archetype (GAE Standard, uses Cloud SDK based maven / gradle plugins)
```

Comment:
We certainly have a lot of these! Do we advertise them anywhere as the way to start your project?